src/ngx_http_lua_ssl_certby.c

```diff
@@ -854,6 +854,47 @@ ngx_http_lua_ffi_ssl_server_name(ngx_http_request_t *r, char **name,
 }
 
 
+int
+ngx_http_lua_ffi_ssl_server_port(ngx_http_request_t *r, unsigned int *port,
+    char **err)
+{
+    ngx_ssl_conn_t          *ssl_conn;
+    ngx_connection_t        *c;
+    ngx_uint_t               p;
```

Comment:
Please use the variable name `port` here instead. `p` usually means pointers, like string pointers.